src/databricks/labs/ucx/progress/install.py

```diff
@@ -0,0 +1,51 @@
+import datetime as dt
+import logging
+from dataclasses import dataclass
+
+from databricks.labs.lsql.backends import SqlBackend
+from databricks.labs.lsql.deployment import SchemaDeployer
+
+
+logger = logging.getLogger(__name__)
+
+
+@dataclass(frozen=True, kw_only=True)
+class WorkflowRun:
```

Comment:
please add a task that runs after all tasks in the MigrationProgress workflow https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/progress/workflows.py#L5

pick the `ctx.named_parameters.get("parent_run_id")` & `job_id` to populate things

https://github.com/databrickslabs/ucx/blob/main/src/databricks/labs/ucx/progress/workflows.py#L5